src/components/apps/AppNavigation.js

```diff
@@ -126,15 +133,40 @@ function AppNavigation(props) {
         } else {
             setCategories(privateCat.categories);
         }
-        handleCategoryChange(privateCat.categories[0]);
     };
 
-    useQuery({
+    const { isFetching } = useQuery({
         queryKey: "getPrivateCategories",
         queryFn: getPrivateCategories,
-        config: { onSuccess: preProcessData, refetchOnWindowFocus: false },
+        config: {
+            onSuccess: preProcessData,
+            onError: () => {
+                //temporary workaround -> should still display community data
+                announce({
+                    text: "Unable to fetch categories",
+                    variant: AnnouncerConstants.ERROR,
+                });
+            },
+            staleTime: Infinity,
+            cacheTime: Infinity,
+        },
     });
 
+    useEffect(() => {
+        setCategoryStatus(isFetching);
+    }, [isFetching, setCategoryStatus]);
```

Comment:
Seems like you could modify `const {isFetching: categoryStatus} = useQuery...` instead of adding this `useEffect`